packages/api/src/external/carequality/command/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -115,7 +115,8 @@ function isValidUrl(url: string | undefined): boolean {
 }
 
 function getCoordinates(address: Address[]): Coordinates | undefined {
-  const orgPosition = address.find(a => a.extension?.url === ORG_POSITION);
+  const addressArray = Array.isArray(address) ? address : [address];
```

Comment:
The complete list of orgs on GDrive have address as a single object, same for CQ staging and production. Not sure how this was working.